Lib/lib2to3/fixes/fix_operator.py

```diff
@@ -1,6 +1,6 @@
 """Fixer for operator functions.
 
-operator.isCallable(obj)       -> hasattr(obj, '__call__')
+operator.isCallable(obj)       -> isinstance(obj, collections.abc.Callable)
```

Comment:
Why not use the built-in `callable`?  It’s been re-added to Python 3 a few versions ago.

Reply:
Since ``callable`` is not available in all 3.x versions, this is probably a safer solution.

Reply:
It depends if 2to3 is meant to convert code to Python 3 all versions, Python 3 currently maintained branches, or specific Python 3.X that shipped that particular 2to3.  Probably a question for python-dev!

Reply:
Should I send this issue to the mailing list?

Reply:
I think so, to clarify the compatibility policy of lib2to3.